config/config.js

```diff
@@ -0,0 +1,28 @@
+require("dotenv").config({ path: __dirname + "/../.env" });
+module.exports = {
+  development: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_DATABASE,
+    host: process.env.DB_HOST,
+    port:process.env.DB_PORT,
+    dialect: "postgres",
+    define: {
+      underscored: true,
+    },
+  },
+  test: {
+    username: "root",
+    password: null,
+    database: "database_test",
+    host: "127.0.0.1",
+    dialect: "mysql",
+  },
+  production: {
+    username: "root",
+    password: null,
+    database: "database_production",
+    host: "127.0.0.1",
+    dialect: "mysql",
+  },
```

Comment:
use environment variable from `.env` for production as well.

Reply:
@raghvendragkmit not resolved yet

Reply:
resolved .env for production